File: stockmarket/stockmarket/models/stock_transaction.py
from .base import Base
from sqlalchemy import Column, Integer, DateTime, String

class StockTransaction(Base):
    __tablename__ = "StockTransaction"
    id = Column(Integer, primary_key=True, autoincrement=True)
    code = Column(String)
    date = Column(DateTime)
    last_transaction = Column(String)
    max_transaction = Column(String)
    min_transaction = Column(String)    
    avg_transaction = Column(String)
    cash_flow_per = Column(String)
    quantity = Column(String)
    cash_flow = Column(String)

File: Домашна 2/tech prototype/backend/models/StockTransaction.js
import { sequelize } from "../config/db-config.js";
import { DataTypes } from "sequelize";

const StockTransaction = sequelize.define(
  "StockTransaction",
  {
    id: {
      type: DataTypes.INTEGER,
      primaryKey: true,
      autoIncrement: true,
    },
    code: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    date: {
      type: DataTypes.DATE,
      allowNull: true,
    },
    last_transaction: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    max_transaction: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    min_transaction: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    avg_transaction: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    cash_flow_per: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    quantity: {
      type: DataTypes.STRING,
      allowNull: true,
    },
    cash_flow: {
      type: DataTypes.STRING,
      allowNull: true,
    },
  },
  {
    tableName: "StockTransaction",
    timestamps: false,
  }
);

export default StockTransaction;
